unison-src/transcripts/update-on-conflict.output.md

```diff
@@ -75,21 +72,27 @@ x = 1 + 2
   do an `add` or `update`, here's how your codebase would
   change:
   
-    ⍟ These new definitions are ok to `add`:
+    ⍟ These names already exist. You can `update` them to your
```

Comment:
It would be nice to indicate in this message that `updating` would resolve conflicts just so the user has full info about what they're about to do, but I think maybe that's tricky to do with the new slurp changes eh?